### DISCUSSION TITLE
conf.ini

### DISCUSSION THREAD

**Discussion by charleslcso** [cloudreve/cloudreve#1501] [Q&A]
I have this:

[Panic]   2022-10-10 21:01:25 无法解析配置文件 '/cloudreve/conf.ini': BOM: read /cloudreve/conf.ini: is a directory

/cloudreve/conf.ini 真的是文件夾。我該如何處理呢？

**Comment by neveler** [comment]
你把他删了，新建一个文件命名为conf.ini就好了……
